index.html

```diff
@@ -424,16 +424,76 @@ <h1 class="title is-1"> Past Sponsors</h1>
       </div>
     </div>
   </section>
-  <section class="section sponsors" id="team">
+
+  <section class="section team" id="team">
     <div class="container has-text-centered">
       <h1 class="title is-1"> Team</h1>
       <div class="row">
-        <img src="img/team/Shivam.jpg" width="30%">
-        <img src="img/team/Suhail.jpg" width="30%">
-        <img src="img/team/bansal.jpeg" width="30%">
+
+        <div class="col-12 col-sm-12 col-md-5 col-lg-5 team-member-box">
+          <div class="row">
+            <div class="col-4 col-sm-5 col-md-4 col-lg-4">
+              <img src="img/team/Shivam.jpg" class="res-img">
+            </div>
+            <div class="col-8 col-sm-7 col-md-8 col-lg-8 ">
+              <h6 class="member-name">NAME</h6>
+              <hr class="card-divider"/>
+              <p class="member-bio">BIO GOES HERE</p>
+              <span class="icon-box">
+                  <a href="#"><i class="fab fa-facebook icon "></i></a>
+                  <a href="#"><i class="fab fa-instagram icon "></i></a>
+                  <a href="#"><i class="fab fa-github-square icon "></i></a>
+                  <a href="#"><i class="fab fa-pinterest icon "></i></a>
+                  <a href="#"><i class="fab fa-linkedin icon "></i></a>
+              </span>
+            </div>
+         </div>
+       </div>
+
+        <div class="offset-md-1 offset-lg-1 col-12 col-sm-12 col-md-5 col-lg-5 team-member-box">
+          <div class="row">
+            <div class="col-4 col-sm-4 col-md-4 col-lg-4">
+              <img src="img/team/Suhail.jpg" class="res-img">
+            </div>
+            <div class="col-8 col-sm-8 col-md-8 col-lg-8 ">
+              <h6 class="member-name">NAME</h6>
+              <hr class="card-divider"/>
+              <p class="member-bio">BIO GOES HERE</p>
+              <span class="icon-box">
+                  <a href="#"><i class="fab fa-facebook icon "></i></a>
+                  <a href="#"><i class="fab fa-instagram icon "></i></a>
+                  <a href="#"><i class="fab fa-github-square icon "></i></a>
+                  <a href="#"><i class="fab fa-pinterest icon "></i></a>
+                  <a href="#"><i class="fab fa-linkedin icon "></i></a>
```

Comment:
What's your review about linkedin profile @suhailsinghbains ?

Reply:
As I don't have an account on Facebook,
I should be having LinkedIn Icon and you both can have Facebook icons, so, that, number of links for each team member is equal. :)

Reply:
@suhailsinghbains linkedin icon is already present so it'll not be replaced with the facebook one, should I remove the fb-icon from your card or add some other link. (mozilla and telegram icons are added in place of instagram and pinterest respectively.

Reply:
@akshat0047 
Yeah, remove fb-icon from my card and remove linkedin icon from other people's card. 
Hope I make sense, feel free to revert :)